app/views.py

```diff
@@ -0,0 +1,143 @@
+
+import os
+from user import User
+from recipes import RecipeCat
+from flask import Flask, render_template, request, session, redirect, url_for
+
+from app import app
+
+app.secret_key = os.urandom(24)
+"""Instantiating objects"""
+newUser = User()
+newRecipeCat = RecipeCat()
+
+@app.route('/')
+def home():
+    return render_template("home.html")
+
+@app.route('/register', methods=['GET', 'POST'])
+def register():
+    """define method to register users"""
+    if request.method == 'POST':
+        username = request.form['username']
+        email = request.form['email']
+        password = request.form['password']
+        cpassword = request.form['cpassword']
+        result = newUser.register(email, username, password, cpassword)
+
+        if result == 1:
+            session['user'] = username
+            message = "Account created sucessfully"
+            return render_template('login.html', data = message)
+        elif result == 6:
+            message = ("please fill all the fields")
+            return render_template('register.html', data = message)
+        elif result == 5:
+            message = ("Special characters not allowed in field name")
+            return render_template('register.html', data=message)
+        elif result == 3:
+            message = ("password do not match")
+            return render_template('register.html', data=message)
+        elif result == 2:
+            error = "email must be a valid email"
+            return render_template('register.html', data=error)
+        elif result == 4:
```

Comment:
Order this if statement 1, 2,3...6 numerically

Reply:
You have not validated if the username exists. So it means I can create a new user with and already existing username